scripts/onboarding/aks/onboarding-msi-azure-policy/README.md

```diff
@@ -0,0 +1,17 @@
+You can create the policy definition using a command like in windows:
+
+```az policy definition create --name "Preview-AKS-Monitoring-Addon-MSI" --display-name "Preview-AKS-Monitoring-Addon-MSI" --mode Indexed --metadata version=1.0.0 category=Kubernetes --rules .\azure-policy.rules.json --params .\azure-policy.parameters.json```
```

Comment:
Is it a preview policy? For built-in policies, this is the versioning guide: https://dev.azure.com/msazure/One/_wiki/wikis/One.wiki/87210/Versioning-Preview-Deprecation-Guidelines 
version=1.0.0-preview.

Reply:
removed